src/OnlineSales/Interfaces/IEmailFromTemplateService.cs

```diff
@@ -0,0 +1,12 @@
+﻿// <copyright file="IEmailFromTemplateService.cs" company="WavePoint Co. Ltd.">
+// Licensed under the MIT license. See LICENSE file in the samples root for full license information.
+// </copyright>
+
+namespace OnlineSales.Interfaces;
+
+public interface IEmailFromTemplateService
```

Comment:
@gsplakmal we need support for scheduleId parameter here.

Reply:
Updated.